docs/self-managed/identity/orchestration-identity/configuration.md

```diff
@@ -0,0 +1,45 @@
+---
+id: configuration
+title: "Configuration"
+description: "Learn about the Identity configuration options available in your Orchestration cluster."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+As a Spring Boot application, Identity supports any standard
+[Spring configuration](https://docs.spring.io/spring-boot/reference/features/external-config.html) method.
+
+<!-- updates must be made to BOTH tables -->
+<Tabs>
+  <TabItem value="env" label="Environment variables" default>
+
+| Environment variable                                  | Description                                                                                       | Default value       |
```

Comment:
💭 These variables are applicable to all components in the orchestration cluster so I think we need to find a central home for these. As per @megglos [suggestion](https://github.com/camunda/documentation-team/issues/465) I wonder if we can create a new area for the Core components? 

It may be that its more important to just get this in place regardless and then adjust as we go, happy to align off PR.

Reply:
Followed up on the related issue, but I think ideally we move forward with this iteration, and then make additional Core-related changes in subsequent PRs: https://github.com/camunda/documentation-team/issues/465#issuecomment-2701824081

Reply:
In favor of having this as a first increment too.